source/_docs/sendgrid.md

```diff
@@ -74,7 +74,8 @@ Support for Drupal 8 is not yet available for the [SMTP Authentication Support](
 Your Drupal application on Pantheon is now set up to send email through SendGrid. Provide an address within the Send Test E-mail configuration field and click **Save configuration** to test.
 
 ### SendGrid API Integration
-The SendGrid Integration module is not supported on Drupal 8 sites at this time. This method can be problematic on Pantheon due to [Composer Manager](https://www.drupal.org/project/composer_manager) (required by [SendGrid Integration](https://www.drupal.org/project/sendgrid_integration)).
+The SendGrid Integration module is not supported on Drupal 8 sites at this time. This method can be problematic on Pantheon due to [Composer Manager](https://www.drupal.org/project/composer_manager) (required by [SendGrid Integration](https://www.drupal.org/project/sendgrid_integration)). This module currently needs a patch to work with Drupal 7, see [this issue](https://www.drupal.org/node/2759433#comment-11359957) for more information.
+
```

Comment:
@dustinleblanc moved to sendgrid_integration section, instead of within the SMTP part of the doc. Do you think the D8 statements are up to date for this module? [Project page](https://www.drupal.org/project/sendgrid_integration) says 
> This module has been included with Drupal 8 core. Refer to this [issue](https://www.drupal.org/node/2676416) for more information.


Thanks for the PR!